pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/BaseClusterIntegrationTestSet.java

```diff
@@ -645,9 +645,11 @@ public String reloadTableAndValidateResponse(String tableName, TableType tableTy
       throws IOException {
     String response =
         sendPostRequest(_controllerRequestURLBuilder.forTableReload(tableName, tableType, forceDownload), null);
+    response = StringEscapeUtils.unescapeJava(response);
     String tableNameWithType = TableNameBuilder.forType(tableType).tableNameWithType(tableName);
     JsonNode tableLevelDetails =
-        JsonUtils.stringToJsonNode(StringEscapeUtils.unescapeJava(response.split(": ")[1])).get(tableNameWithType);
+        JsonUtils.stringToJsonNode(response.substring(
```

Comment:
This is very hard to understand. I think we might need to parse twice? Something like:
```
JsonNode responseJson = JsonUtils.stringToJsonNode(response);
JsonNode tableLevelDetails = JsonUtils.stringToJsonNode(responseJson.get("status").asText()).get(tableNameWithType);
```

Reply:
Hi @Jackie-Jiang the the value corresponding to the "status" is another json string. So parsing it into json fails because the value is wrapped in "". I just did the substring logic to get the wrapped string. Thanks!

Reply:
The json string is encoded as the value field of the top level json, so parsing the json twice should be able to extract it. See the code example above

Reply:
Got it thanks, I did this, it seems the 6 tests fail. The result is wrapped in "", so the `JsonUtils.stringToJsonNode` fails parsing it, to fix the test we would have to extract the value and then parse it. What do you think. Thanks!

Reply:
The problem is the unnecessary unescape. Applied a commit and let's see if it fixes the problem

Reply:
Oh got it thank you so much!